pipelines/cemba/cemba_methylcseq/README.md

```diff
@@ -1,40 +1,184 @@
+- [Set-up](#set-up)
+  * [CEMBA Installation and Requirements](#cemba-installation-and-requirements)
+  * [Inputs](#inputs)
+    + [Sample data input](#sample-data-input)
+    + [Additional Inputs](#additional-inputs)
+- [CEMBA Tasks and Tools](#cemba-tasks-and-tools)
+  * [CEMBA Task Summary](#cemba-task-summary)
+- [Outputs](#outputs)
+- [Versioning](#versioning)
+- [Have Suggestions?](#have-suggestions)
+
+
+
+# CEMBA Pipeline Overview
+
+<img src="CEMBA.png" width="750">
+
+## Introduction to the CEMBA Workflow
+
+CEMBA is a pipeline developed by the Data Coordination Platform (DCP) of the [Human Cell Atlas (HCA) Project](https://data.humancellatlas.org/) that supports processing of multiplexed single-nuclei bisulfite sequencing data. It is an alignment and methylated base calling pipeline that trims adaptors, attaches cell barcodes, aligns reads to the genome, filters reads based on quality and creates a VCF with methylation-site coverage. 
+
+*Interested in using the pipeline for your publication? See the [“CEMBA publication methods”](CEMBA.methods.md) for a generic "methods" style description of the pipeline.*
+
+
+## Quick Start Table
+
+| Pipeline Features | Description | Source |
+|-------------------|---------------------------------------------------------------|-----------------------|
+| Assay Type | Single-nucleus methylcytosine bisulfite sequencing | [Luo et al. 2017](https://science.sciencemag.org/content/357/6351/600/tab-pdf)
+| Overall Workflow  | Aligns reads and calls bases | Code available from [Github](CEMBA.wdl) |
+| Workflow Language | WDL 1.0 | [openWDL](https://github.com/openwdl/wdl) |
+| Genomic Reference Sequence| GRCH38 and GRCM38  |  [GENCODE](https://www.gencodegenes.org/)  |
+| Aligner  | BISMARK v0.21.0 with  --bowtie2 | [Bismark](https://www.bioinformatics.babraham.ac.uk/projects/bismark/) |
+| Varariant Caller | GATK 4.1.2.0 | [GATK 4.1.2.0](https://gatk.broadinstitute.org/hc/en-us)                          
+| Data Input File Format | File format in which sequencing data is provided | [Zipped FASTQs (.fastq.gz)](https://support.illumina.com/bulletins/2016/04/fastq-files-explained.html) |                     
+| Data Output File Format | File formats in which CEMBA output is provided | [BAM](http://samtools.github.io/hts-specs/), [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf) |
+
+# Set-up
+
+## CEMBA Installation and Requirements
+The CEMBA pipeline code is written in the [Workflow Description Language (WDL)](https://github.com/broadinstitute/warp) and can be downloaded by cloning the GitHub repository [WARP](https://github.com/broadinstitute/warp). For the latest release of CEMBA, please see the release tags prefixed with "CEMBA" [here](https://github.com/broadinstitute/warp/tags). CEMBA can be deployed using [Cromwell](https://github.com/broadinstitute/cromwell), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. 
```

Comment:
Do we usually note if the pipeline is available in Terra in this section?

Reply:
We can note that- is this the pipeline you'd like to link to: https://app.terra.bio/#workspaces/brain-initiative-bcdc/Methyl-c-seq_Pipeline?

I can't access the one in the JIRA ticket (I can access the dev one), so just want to confirm the correct one.

Reply:
Yep https://app.terra.bio/#workspaces/brain-initiative-bcdc/Methyl-c-seq_Pipeline

That's the one.

Reply:
Added!